CRM/Member/BAO/Query.php

```diff
@@ -165,29 +165,36 @@ public static function whereClauseSingle(&$values, &$query) {
     switch ($name) {
       case 'member_join_date_low':
       case 'member_join_date_high':
+      case 'membership_join_date_low':
```

Comment:
@seamuslee001 I'm not keen on this - it leaves the deprecated code 'looking useful' - I'd rather separate out the 2 & have deprecation notices in the obsolete code.

We could handle the 'good fields' like this https://github.com/civicrm/civicrm-core/pull/15191